includes/filerepo/file/LocalFile.php

```diff
@@ -236,15 +236,6 @@ function saveToCache() {
 			}
 		}
 
-		/* Wikia change begin */
-		// If there's no timestamp with this file don't cache it, its a soon
-		// to be uploaded file that hasn't been fully saved yet.
-		if ( empty($cache['timestamp']) ) {
-			$wgMemc->delete( $key );
-			return;
-		}
-		/* Wikia change end */
-
 		$wgMemc->set( $key, $cache, 60 * 60 * 24 * 7 ); // A week
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)